client/src/modules/stock/lots-duplicates/duplicate-lots.js

```diff
@@ -0,0 +1,165 @@
+angular.module('bhima.controllers')
+  .controller('DuplicateLotsController', DuplicateLotsController);
+
+DuplicateLotsController.$inject = [
+  'LotService', 'NotifyService', 'uiGridConstants', 'StockModalService',
+  'SessionService', 'GridStateService', '$state', 'GridColumnService',
+];
+
+/**
+ * Stock Inventory Controller
+ * This module is a registry page for stock inventories
+ */
+function DuplicateLotsController(
+  Lots, Notify, uiGridConstants, Modal,
+  Session, GridState, $state, Columns,
+) {
+  const vm = this;
+  const cacheKey = 'duplicate-lots-grid';
+  vm.enterprise = Session.enterprise;
+
+  // const duplicateLotsFilters = Stock.filter.inventory;
```

Comment:
Let's just remove this.

Reply:
I wanted this to sort next to the 'lots' file folder uner 'stock', hence the name.  I plan to rename the files to 'lots-duplicates.*' to match the directory name.   'duplicate-lots' would be more readable, but would make it a bit harder to find with the file explorer.

Reply:
Removed